app/scripts/views/sign_up.js

```diff
@@ -218,6 +222,15 @@ function (Cocktail, _, p, BaseView, FormView, Template, AuthErrors, mailcheck,
       return this._isMailcheckEnabledValue;
     },
 
+    passwordCheck: function () {
+      var self = this;
+      var password = this.getElementValue('.password');
+      this.passwordcheck(password, function (msg) {
+        self.logScreenEvent(msg);
```

Comment:
I forgot, you'll also have to add any new errors and events to the [client-metrics document](https://github.com/mozilla/fxa-content-server/blob/master/docs/client-metrics.md#signup) under the signup page.